### ISSUE TITLE
Feature request: modify "height" of hilight areas to correspond to cladelabel bar

### ISSUE THREAD

**Issue by ferroao** [GuangchuangYu/ggtree#142]
Hilight areas seem of appropriate size for non-circular plots, though the height of the area is different from the length of the cladelabel bar.
p <- ggtree(tree) + xlim(NA, 6)
p+geom_cladelabel(node=45, label="test label") +
  geom_cladelabel(node=34, label="another clade") + geom_hilight(34, fill="grey70")

However, when plotted circular, it would be desirable that the "height" of the area be the same as the length of the cladelabel bar, this is specially necessary for dense plots with very near clades.
p <- ggtree(tree, layout = "circular") + xlim(NA, 6)
p+geom_cladelabel(node=45, label="test label") +
  geom_cladelabel(node=34, label="another clade") + geom_hilight(34, fill="grey70")

![rplotpngcircular](https://user-images.githubusercontent.com/14240133/28544826-fd8ed324-709a-11e7-9956-00dda4b7d320.png)

**Comment by GuangchuangYu** [comment]
```r
args(geom_hilight)
function (node, fill = "steelblue", alpha = 0.5, extend = 0,
    extendto = NULL)
```

see the man file of the layer.

**Comment by GuangchuangYu** [comment]
As [two of your feature request](https://github.com/GuangchuangYu/ggtree/issues/141) are all available and well documented, I suggest you to read the [guide](https://guangchuangyu.github.io/2016/07/how-to-bug-author/) before posting.

If you ignore the [github template](https://github.com/GuangchuangYu/ggtree/issues/new), I will ignore you too.

**Comment by ferroao** [comment]
The length of the cladelabel bar is different to the same dimension (which I called height, but is height only in non-circular plots) is not modifiable by extend or extendto, see picture.

**Comment by GuangchuangYu** [comment]
```r
p <- ggtree(tree, layout = "circular") + xlim(NA, 6)
p + geom_cladelabel(node=45, label="test label") +
    geom_cladelabel(node=34, label="another clade", extend=0.5) + 
    geom_hilight(34, fill="grey70")
```

now `extend` parameter is added in the devel branch.

**Comment by ferroao** [comment]
I am sorry to write again, I made the homework  and I saw the bar now extends to the length of the area. So now they have the same "length". It looks good thx, however, my initial request referred to do the opposite, that is, diminish the length of the area (which i called height initially) to fit to the cladelabel length. Currently not the extend, not the extendto parameters of the geom_hilight affect that dimension of the area. thx in advance.
